### PR TITLE
gedit: add Spanish translation

### PR DESCRIPTION
**Pull request by advy99**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.es/linux/gedit.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# gedit
+
+> Editor de texto del proyecto GNOME.
+> Más información: <https://help.gnome.org/users/gedit/stable/>.
+
+- Abrir un fichero de texto:
```
Comment (navarroaxel) [inline pages.es/linux/gedit.md:6]: For Romance languages, the present indicative fits better.
```suggestion
- Abre un archivo de texto:
```
We use `archivo` over `fichero`.

File: pages.es/linux/gedit.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`gedit {{path/to/file}}`
```
Comment (navarroaxel) [inline pages.es/linux/gedit.md:8]: Please translate the placeholders for get this easier for non-English speakers.
```suggestion
`gedit {{ruta/al/archivo}}`
```

File: pages.es/linux/gedit.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Abrir varios ficheros de texto:
+
+`gedit {{file1 file2 ...}}`
```
Comment (navarroaxel) [inline pages.es/linux/gedit.md:12]:
```suggestion
`gedit {{archivo1 archivo2 ...}}`
```

File: pages.es/linux/gedit.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- Abrir un fichero de texto con una codificación específica:
+
+`gedit --encoding={{UTF-8}} {{path/to/file}}`
```
Comment (navarroaxel) [inline pages.es/linux/gedit.md:16]:
```suggestion
`gedit --encoding={{UTF-8}} {{ruta/al/archivo}}`
```

File: pages.es/linux/gedit.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+- Abrir un fichero de texto:
+
+`gedit {{path/to/file}}`
+
+- Abrir varios ficheros de texto:
+
+`gedit {{file1 file2 ...}}`
+
+- Abrir un fichero de texto con una codificación específica:
```
Comment (navarroaxel) [inline pages.es/linux/gedit.md:14]:
```suggestion
- Abre un archivo de texto con una codificación específica:
```

File: pages.es/linux/gedit.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# gedit
+
+> Editor de texto del proyecto GNOME.
+> Más información: <https://help.gnome.org/users/gedit/stable/>.
+
+- Abrir un fichero de texto:
+
+`gedit {{path/to/file}}`
+
+- Abrir varios ficheros de texto:
```
Comment (navarroaxel) [inline pages.es/linux/gedit.md:10]:
```suggestion
- Abre varios archivos de texto:
```

File: pages.es/linux/gedit.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+`gedit {{file1 file2 ...}}`
+
+- Abrir un fichero de texto con una codificación específica:
+
+`gedit --encoding={{UTF-8}} {{path/to/file}}`
+
+- Mostrar una lista de las codificaciones disponibles:
```
Comment (navarroaxel) [inline pages.es/linux/gedit.md:18]:
```suggestion
- Muestra una lista de las codificaciones disponibles:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @advy99, welcome and thanks for your submission. I left some comments.

**Comment by advy99** [comment]
Hi! I just made a commit with the required changes

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**